### PR TITLE
fix: warning for inactive account

### PR DESCRIPTION
**Pull request by sagzy**
The warning is now shown directly on the dashboard homepage (maps#show), with an explicit CTA to subscribe to one of the plans (instead of "something is wrong with your account"
Screenshot 2021-12-30 at 16 40 50
)

### CODE REVIEW

File: app/controllers/concerns/inactive_notifier.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module InactiveNotifier
```
Comment (shafy) [inline app/controllers/concerns/inactive_notifier.rb:1]: Should we stick with the Rails naming convention and call it `InactiveNotifiable`? I guess would also need to update the other concerns `TrialNotfiable`. Up to you.

**Review by shafy — APPROVED** [review]

**Comment by shafy** [comment]
Cool! This doesn't put the map in an inactive state though, right?

File: app/controllers/concerns/inactive_notifier.rb (hunk 1 above)
Comment (sagzy) [reply app/controllers/concerns/inactive_notifier.rb]: Yes good point, I'll rename it

**Comment by sagzy** [comment]
> Cool! This doesn't put the map in an inactive state though, right?

No it doesn't, only improves the warning mechanism. I reckon we can solve this problem for real when it becomes a real problem

**Pull request merged**